core/workspace_svg.js

```diff
@@ -1815,11 +1816,16 @@ Blockly.WorkspaceSvg.prototype.showContextMenu = function(e) {
 
 /**
  * Modify the block tree on the existing toolbox.
- * @param {Node|string} tree DOM tree of blocks, or text representation of same.
+ * @param {Node|Array.<Blockly.utils.toolbox.ToolboxInfo>|Array.<Node>|string} toolboxDef
```

Comment:
Since this type (the combination of all of the others) is popping up all over the place, consider making a typedef for it.

Reply:
done